src/app/code/community/Wf/Bananastand/Helper/Data.php

```diff
@@ -143,5 +143,85 @@ public function isDebugMode()
         return Mage::getStoreConfigFlag('banana/general/debug_mode');
     }
 
+    /**
+     * @return json - The contents of the cart as a json string.
+     */
+    public function getCartJson()
+    {
+        $cartItems = Mage::getModel('checkout/cart')->getItems();
+        if (empty($cartItems)) {
+          return "[]";
+        }
+        return json_encode($cartItems->getData());
+    }
+
+    /**
+     * Format a mage order object into JSON for sending to VIA js api
+     * @param Mage_Sales_Model_Order $order
+     * @return JSON
+     */
+    public function jsonifyOrder($order)
+    {
+        $items = array();
+
+        foreach ($order->getAllItems() as $item) {
+            $product = $item->getProduct();
+            $items[] = array(
+                'name'     => $item->getName(),
+                'total'    => $item->getPrice(),
+                'quantity' => $item->getQtyOrdered(),
+                'product'  => array(
+                    'id'            => $product->getId(),
+                    'name'          => $product->getName(),
+                    'status'        => $product->getStatus(),
+                    'in_stock'      => $product->isInStock(),
+                    'url'           => $product->getProductUrl(),
+                    'thumbnail_url' => $product->getThumbnailUrl(),
+                    'platform_data' => $product->getData()
```

Comment:
This can actually cause issues because sometimes the getData() contains models that can't be serialized... Their data system is sort of annoying for this reason... so let's just leave it out for now.